src/core/Akka.Tests/Pattern/BackoffSupervisorSpec.cs

```diff
@@ -209,7 +209,7 @@ public void BackoffSupervisor_must_support_manual_reset()
                 });
 
                 // TODO: this Thread.Sleep should be removed
-                Thread.Sleep(1000);
+                Thread.Sleep(500);
```

Comment:
This test should work without any Thread.Sleep, but it does not. In debug mode all works fine